deploy/crds/operators_v1_catalogsourceconfig_crd.yaml

```diff
@@ -39,6 +39,9 @@ spec:
           - targetNamespace
           - packages
           properties:
+            sourceName:
+              type: string
+              description: The source that the operator originates from
```

Comment:
Can we be more expressive here? Personally I think we should be explicit in this description that this should be the name of the CatalogSource/OperatorSource, otherwise users will be confused by the documentation. In the future we can always change the description if the source changes without changing the name itself.

Reply:
You can call out that the source is only `OperatorSources` here for now

Reply:
I will update the description:
>The name of the OperatorSource that the operator originates from